src/libraries/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -11,9 +11,14 @@ namespace System.Diagnostics.Tests
     public class ProcessStandardConsoleTests : ProcessTestBase
     {
         [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer), nameof(PlatformDetection.IsNotWindowsServerCore), nameof(PlatformDetection.IsNotWindowsIoTCore))]
         public void TestChangesInConsoleEncoding()
         {
+            // These OS variants don't work with setting a default Encoding.
+            if (PlatformDetection.IsNotWindowsNanoServer || PlatformDetection.IsNotWindowsServerCore || PlatformDetection.IsNotWindowsIoTCore)
```

Comment:
Shouldn't this be `&&` not `||` (or remove the `Not`'s)? 

Otherwise it's `if (IsNotAnOrange || IsNotABicycle)`

Reply:
I think you need to remove the `Not`'s

Reply:
yeah that part was wrong. But I removed it anyways as Santi took care of this in another PR.